raddb/mods-config/perl/packetfence.pm

```diff
@@ -339,16 +339,10 @@ sub accounting {
 
         my $config = _get_rpc_config();
         my $data;
+        $data = send_msgpack_notification($config, "handle_accounting_metadata", \%RAD_REQUEST);
         if ($RAD_REQUEST{'Acct-Status-Type'} eq 'Stop' || $RAD_REQUEST{'Acct-Status-Type'} eq 'Interim-Update') {
             $data = send_rpc_request($config, "radius_accounting", \%RAD_REQUEST);
-        } elsif ($RAD_REQUEST{'Acct-Status-Type'} eq 'Start') {
-            #
-            # Updating location log in on initial ('Start') accounting run.
-            #
-            $data = send_rpc_request($config, "radius_update_locationlog", \%RAD_REQUEST);
-        }
-        # Tracking IP address.
-        send_rpc_request($config, "update_iplog", {mac => $mac, ip => $RAD_REQUEST{'Framed-IP-Address'}, source => "accounting"}) if ($RAD_REQUEST{'Framed-IP-Address'} );
+        } 
```

Comment:
I don't think so because we want to send it for every type that exists so we can extract out metadata out of every packet.

I initially made this piece of code for accounting fingerprinting and its present in every packet so we want the metadata to be handled on every packet